myia/frontends/pytorch.py

```diff
@@ -102,12 +131,16 @@ def mod_sub(self, x):
 # TODO: should all of these actually be blacklisted (not used).
 # Curently blacklists all constructors except '_parameters' and '_modules'.
 # 'training' should probably be removed from blacklist in next PR.
+"""
 blacklist = ('_backend', '_buffers', '_backward_hooks', '_forward_hooks',
              '_forward_pre_hooks', '_state_dict_hooks',
              '_load_state_dict_pre_hooks',
 
              'training'
              )
+             """
+blacklist = set(dir(torch.nn.Module()) + ["__constants__"])
+blacklist.add('reset_parameters')
```

Comment:
You're adding two entries to the blacklist in two different ways (+ and .add), which is not very consistent.